src/database/datatools/nomass_materials.py

```
@@ -0,0 +1,146 @@
+                             roughness: str,
+                             thermal_resistance: float,
+                             thermal_absorptance: Optional[float] = None,
+                             solar_absorptance: Optional[float] = None,
+                             visible_absorptance: Optional[float] = None) -> None:
+    
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "no_mass_materials"
+
+    sql = f"INSERT INTO {table_name} (name, latitude, longitude, architecture_type, roughness, thermal_resistance, thermal_absorptance, solar_absorptance, visible_absorptance, datetime) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)